cypress/e2e/oss/onboarding.test.ts

```diff
@@ -213,10 +209,9 @@ describe('Onboarding', () => {
     // check password too short
     cy.getByTestID('form--element-error').should('contain.text', '8')
 
-    cy.getByTestID('next')
-      .should('be.disabled')
-      .children('.cf-button--label')
-      .contains('Continue')
+    cy.getByTestID('next').should('be.disabled')
+
+    cy.contains('CONTINUE')
```

Comment:
this is a smell to me - what element are we checking contains "CONTINUE"?

Reply:
It's a button in the onboarding form for OSS; That button has the text content 'CONTINUE'. I am not sure why the original test writer was checking that button's text content like million times in the test. 

I refactored this test to be more coherent by using .contains on the actual button

Reply:
perfect!